tasks/infra_ops.py

```diff
@@ -33,7 +33,11 @@ def load_infrastructure_schema(
         compose_files_cmd = build_compose_files_cmd(database=database, namespace=namespace)
         compose_cmd = get_compose_cmd(namespace=namespace)
         base_cmd = f"{get_env_vars(context, namespace=namespace)} {compose_cmd} {compose_files_cmd} -p {BUILD_NAME}"
-        command = f"{base_cmd} run {SERVICE_WORKER_NAME} infrahubctl schema load {target}"
+        command_schema = f"{base_cmd} run {SERVICE_WORKER_NAME} infrahubctl schema load {target}"
         if add_wait:
-            command += " --wait 30"
-        execute_command(context=context, command=command)
+            command_schema += " --wait 30"
+        execute_command(context=context, command=command_schema)
+
+        if namespace == Namespace.DEV:
+            command_menu = f"{base_cmd} run {SERVICE_WORKER_NAME} infrahubctl menu load {target}_menu.yaml"
```

Comment:
I'd change the value of '{target}` so that the default value is `models/base_menu.yaml` if we want to override this from the command line I don't think it adds much value to force us to point to a file with the suffix `_menu.yaml` instead of specifying the full path.

Also tiny comment about the name I think every other file under ./models is called `.yml` instead of `.yaml`.